local-macros/src/lib.rs

```diff
@@ -50,7 +50,7 @@ impl AstTypeArgs {
     }
 }
 
-fn get_struct_interface_impl(
+fn get_ast_struct_interface_impl(
```

Comment:
I tried moving the `ast_type` procedural macro implementation into a nested module but it told me that the `#[proc_macro_attribute]` attribute can only be used in the crate root

Reply:
Huh, that's a weird restriction.